task3/src/Components/Utils/ErrorBoundar.jsx

```diff
@@ -0,0 +1,18 @@
+import  React, {Component } from 'react' ;
```

Comment:
Why this file is jsx and others are tsx ?

Reply:
Yep, I have refactored and use typescript